index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Potfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <nav class="nav">
+            <img class="img" src="images/image_geometry_1.svg" alt="clip art">
+            <div class="humburger">
+                <span class="line"></span>
+                <span class="line"></span>
+                <span class="line"></span>
+            </div>
+            <div class="nav__link hide">
+                <a href="index.html">Hello</a>
+                <a href="index.html">Portfolio</a>
+                <a href="about.html">About</a>
+                <a href="contact.html">Contact</a>
+            </div>
+        </nav>
+    </header>
+    <section class="section">
+        <div class="info">
+            <h1 class="name">Emmanuel <br> Tofa</h1>
+            <p class="skills1">Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work.</p>
+            <p class="skills2">If you like what you see and have a project you need coded, don't hesitate to contact me.</p>                
+        </div>
+        <button class="collab">start collaboration</button>
+        <div class="banner"><img src="images/image_geometry_2.png" alt="image"></div>
```

Comment:
- Good job done so far, but since this image is not part of the main content of the page, it is advisable you add it as a background image using CSS. Please fix this 🙏

Reply:
> * Good job done so far, but since this image is not part of the main content of the page, it is advisable you add it as a background image using CSS. Please fix this pray

- [ ] As the previous reviewer suggested, kindly implement it. :+1:

Reply:
kindly the changes mentioned by the previous reviewers are required you need to apply them , this image should be used as a background image of this container.